app/controllers/state_file/questions/md_pension_exclusion_offboarding_controller.rb

```diff
@@ -3,7 +3,7 @@ module Questions
     class MdPensionExclusionOffboardingController < QuestionsController
       include OtherOptionsLinksConcern
       def self.show?(intake)
-        Flipper.enabled?(:show_retirement_ui) && intake.filing_status_mfj? && intake.state_file1099_rs.present? && intake.has_filer_under_65? && intake.filer_disabled? && intake.no_proof_of_disability_submitted?
+        Flipper.enabled?(:show_retirement_ui) && intake.filing_status_mfj? && intake.state_file1099_rs.present? && intake.could_qualify_for_pension_exclusion? && intake.filer_disabled? && intake.no_proof_of_disability_submitted?
```

Comment:
[pebble] with the new logic you can remove `intake.state_file1099_rs.present?`

Reply:
it’s a bit confusing that this could refer to both filers

Reply:
good catch!

Reply:
I could see an argument for it being confusing but I think it's better than having multiple variables in the show page for what is essentially the same thing.

Reply:
ah, i just meant renaming it, not creating a new variable.